services/wallet/router/pathprocessor/constants.go

```diff
@@ -32,3 +32,11 @@ const (
 	ProcessorENSPublicKeyName = "ENSPublicKey"
 	ProcessorStickersBuyName  = "StickersBuy"
 )
+
+func IsProcessorBridge(name string) bool {
```

Comment:
nit: make it a type and make it a function of the type

Reply:
yeah, considered it but I will get a thousand line changes and I wanted to merge this fix sooner :D
I'll do it on a separate PR if that's fine